openstack-upgrade bionic-train hit strange KeyError '2.0'

Probably due to a recent change; will need to dig into it.

```
2020-11-17 12:41:06 [INFO] Running script upgrade_openstack.py
2020-11-17 12:41:40 [WARNING] No debug log matching debug-logs found. Using default.
2020-11-17 12:41:41 [ERROR] 2020-11-17 12:41:08 [WARNING] get_full_juju_status from zaza.openstack.utilities.juju is deprecated. Please use the equivalent from zaza.utilities.juju
2020-11-17 12:41:08 [WARNING] unknown delta type: id
2020-11-17 12:41:08 [WARNING] get_application_status from zaza.openstack.utilities.juju is deprecated. Please use the equivalent from zaza.utilities.juju
2020-11-17 12:41:08 [WARNING] unknown delta type: id

(... previous two lines repeat a lot)

2020-11-17 12:41:21 [WARNING] get_application_status from zaza.openstack.utilities.juju is deprecated. Please use the equivalent from zaza.utilities.juju
2020-11-17 12:41:21 [WARNING] unknown delta type: id

(... previous line repeats a lot)

2020-11-17 12:41:39 [WARNING] unknown delta type: id
/var/lib/jenkins/tools/0/charm-test-infra/.tox/clients/lib/python3.5/site-packages/paramiko/transport.py:33: CryptographyDeprecationWarning: Python 3.5 support will be dropped in the next release of cryptography. Please upgrade your Python.
  from cryptography.hazmat.backends import default_backend
Traceback (most recent call last):
  File "/tmp/tmp.l8oltBv8ps/mojo-openstack-specs/bionic/osci-mojo/spec/specs/full_stack/next_openstack_upgrade/train/upgrade_openstack.py", line 153, in <module>
    sys.exit(main(sys.argv))
  File "/tmp/tmp.l8oltBv8ps/mojo-openstack-specs/bionic/osci-mojo/spec/specs/full_stack/next_openstack_upgrade/train/upgrade_openstack.py", line 120, in main
    principle_services)
  File "/var/lib/jenkins/tools/0/charm-test-infra/.tox/clients/lib/python3.5/site-packages/zaza/openstack/utilities/openstack.py", line 1543, in get_current_os_versions
    get_os_code_info(application['type']['pkg'], version))
  File "/var/lib/jenkins/tools/0/charm-test-infra/.tox/clients/lib/python3.5/site-packages/zaza/openstack/utilities/openstack.py", line 1521, in get_os_code_info
    return OPENSTACK_CODENAMES[vers]
KeyError: '2.0'
```

Proposal: scan code and see what is going on; alternative: deploy manually and see what is happening.

--- a/zaza/openstack/utilities/openstack.py
+++ b/zaza/openstack/utilities/openstack.py
@@ -1493,7 +1493,6 @@ def get_os_code_info(package, pkg_version):
     :returns: Codename for package
     :rtype: string
     """
-    # {'code_num': entry, 'code_name': OPENSTACK_CODENAMES[entry]}
     # Remove epoch if it exists
     if ':' in pkg_version:
         pkg_version = pkg_version.split(':')[1:][0]

--- a/zaza/openstack/utilities/os_versions.py
+++ b/zaza/openstack/utilities/os_versions.py
@@ -256,4 +256,9 @@ PACKAGE_CODENAMES = {
         ('14', 'train'),   # nautilus
         ('15', 'ussuri'),  # octopus
     ]),
+    'placement-common': OrderedDict([
+        ('2', 'train'),
+        ('3', 'ussuri'),
+        ('4', 'victoria'),
+    ]),
 }

